src/main/java/com/minecolonies/colony/ColonyManager.java

```diff
@@ -681,6 +680,25 @@ public static IMessage handleColonyViewCitizensMessage(int colonyId, int citizen
     }
 
     /**
+     * Returns result of {@link ColonyView#handleColonyViewWorkOrderMessage(ByteBuf)} (int, ByteBuf)} if {@link #getColonyView(int)}
+     * gives a not-null result. If {@link #getColonyView(int)} is null, returns null
+     *
+     * @param colonyId      ID of the colony
+     * @param buf           {@link ByteBuf} with colony data
+     * @return              result of {@link ColonyView#handleColonyViewWorkOrderMessage(ByteBuf)} or null
+     */
+    public static IMessage handleColonyViewWorkOrderMessage(int colonyId, ByteBuf buf)
+    {
+        ColonyView view = getColonyView(colonyId);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'view' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)